App/build.gradle

```diff
@@ -5,6 +5,7 @@ android {
     buildToolsVersion '22.0.1'
     lintOptions {
         abortOnError true
+        disable "MissingTranslation"
```

Comment:
Instead of this, we could mark the key as non-translatable. May using donottranslate.xml?